go.mod

```diff
@@ -21,3 +21,5 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/juju/gomaasapi/v2 v2.2.0 => github.com/skatsaounis/gomaasapi/v2 v2.1.1-0.20240502133513-01fd83950b5d
```

Comment:
This replace directive and its consequences on `go.sum` will be removed when dependent PR is merged